easybuild/easyconfigs/l/libXt/libXt-1.1.4-intel-2015a-Python-2.7.10.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'libXt'
+version = '1.1.4'
+versionsuffix = '-Python-2.7.10'
+
+homepage = "http://www.freedesktop.org/wiki/Software/xlibs"
+description = """libXt provides the X Toolkit Intrinsics, an abstract widget library upon which other toolkits are
+ based. Xt is the basis for many toolkits, including the Athena widgets (Xaw), and LessTif (a Motif implementation)."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'optarch': True}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [XORG_LIB_SOURCE]
+
+libx11ver = '1.6.3'
```

Comment:
not needed (only used in one place), so just collapse into `libX11` dep spec line?